packages/solid-repl/repl/main.css

```diff
@@ -29,6 +29,10 @@ div[contenteditable='true']:focus {
   color-scheme: dark;
 }
 
+div.view-lines.monaco-mouse-cursor-text {
```

Comment:
The font of the editor was changed to a this otherwise, so this lines forces the font back to the one used in the current playground.

<img width="529" alt="Screenshot 2024-03-20 at 13 12 34" src="https://github.com/solidjs/solid-playground/assets/74932975/cc88ec51-13eb-4ed1-98e3-b71ceac9f983">

Reply:
It should stay looking like this

<img width="656" alt="Screenshot 2024-03-20 at 14 10 55" src="https://github.com/solidjs/solid-playground/assets/74932975/7265df93-0aad-43c8-85b3-8f4e3007fc14">

Reply:
Hmm, I don't see the font having been changed to Courier anywhere else (such as the monaco playground)... I'm suspicious that means this PR broke loading of css somewhere

Reply:
I've narrowed this down further to a symptom introduced between version 0.45.0 where it's not present and 0.46.0+ where it's present.

I've successfully updated to 0.47.0 in other projects, like the solid-site Tutorial here https://github.com/solidjs/solid-site/pull/530 without seeing a change in font.

Reply:
I figured how to do this without any tricks, but simply by setting the fontFamily property in the monacoEditor.create().